guiEditor/src/components/sceneExplorer/sceneExplorer.scss

```diff
@@ -215,20 +215,40 @@
         user-select: none;         
         display: grid;
         grid-template-columns: 20px 1fr;
+        border: 2px solid transparent;
+        &.hover {
+            border: 2px solid #527089;

```

Comment:
For future reference - are these colors part of a palette? any way of referencing them from there?

Reply:
Yea. They are on this issue: https://github.com/BabylonJS/ThePirateCove/issues/17